README.md

```diff
@@ -460,6 +460,18 @@ Advanced topics are available in [the official docs](https://www.apollographql.c
 * [no-runtime.md](https://www.apollographql.com/docs/android/advanced/no-runtime/) 
 * [subscriptions.md](https://www.apollographql.com/docs/android/advanced/subscriptions/)
 
+
+## Changelog
+[Read about the latest changes to the library](https://github.com/apollographql/apollo-android/releases)
+
+## Contributing
+
+This project is being developed using IntelliJ IDEA or Android Studio. With [introduction of Kotlin Multiplatform](https://github.com/apollographql/apollo-android/blob/master/apollo-api/build.gradle.kts#L10-L21), build may require installation of xcodebuild tools and Xcode 11.
+
+It is recommended to import/open `composite` instead of root folder since that includes `apollo-integration` tests and all sample projects under `samples` folder by using Gradle Composite builds.
```

Comment:
Maybe move the 2 paragraphs above to `Contributing.md` ? And just keep the link in the README ?

This way, people going straight to `Contributing.md` have the full context ?

Reply:
I have the feeling that people look at the bottom of Readme too for this.

What about this: move the first paragraph. Keep 2nd in both since it is not obvious. Couple of times I got the feedback about not being able to import samples

Reply:
Sounds good to me. For what it's worth, I added a [README.md](https://github.com/apollographql/apollo-android/blob/master/samples/README.md) in the `samples` directory to indicate that as well so we hopefully should be pretty covered now.